### ISSUE TITLE
[Feature request] [Discord messages] New button to enable/disable {instanceName} in start of message

### ISSUE THREAD

**Issue by mula-bb** [taniman/profit-trailer-enhancements#912]
Highly appreciated to make it possible to fully customize discord messages with a toggle `Enable` / `Disable` to 'Show {instanceName} info in message?'

This could be fairly easy to implement.

Currently PT sends out a message to discord in this format:
```
  [{instanceName}]
  [Configure a message with parameters as always] 
```
### **Use case:** 
Because `{instanceName}` is present at the start of the message, this is interfering with some custom discordbot and cannot properly 
read the message to perform another action. 
For example: Alphabot (free charting bot for discord, really great, can recommend to add this to PT discord as well) cannot generate a chart when discord receives a message from PT, because `{instanceName}` is in the way.
![afbeelding](https://user-images.githubusercontent.com/44127181/151717839-b5cd2fa9-300c-4b0b-a0f9-d11e69100223.png)

New improved suggestion discord messaging `{instanceName}` TOGGLE enabled:
```
  [{instanceName}] 
  [Configure a message with parameters as always] 
```

New improved suggestion discord messaging `{instanceName}` TOGGLE disabled:
```
  [Configure a message with parameters as always] 
```

### **Result:**
If `{instanceName}` was optional with a `Enable`/ `Disable` toggle, below would be the result
 - Alphabot can now read the message
 - Alphabot can now directly post a chart of the buy or sell moment, helping you see and understand your buy / sell history
 - Other helpful custom automations can be created / built upon
![afbeelding](https://user-images.githubusercontent.com/44127181/151363313-d890c0cb-9e60-48a8-9f4a-574b1061a2f0.png)

### **Additional:**
Enable more parameters to be sent from PT to Discord, for example:
- Separate customizable message when trailing (buy or sell) of a coin starts: `TrailSignal` --> Discord message

**Use Case**
If build in, you can now see charts from Alphabot 'on the go' in your own Discord server where your config would start trailing a coin for buy or sell

**Comment by mula-bb** [comment]
Yes, the screenshots are the result of Advanced Notifications enabled, with a custom message.
Currently PT sends out the [InstanceName] in the beginning of the message, which is interfering with other applications like Alphabot.

With the change to enable/disable toggle [InstanceName] we could build custom stuff like the above example.

If you need more clarification just let me know.

**Comment by taniman** [comment]
Next release has a fix for this (1.5.9), So if you leave out instanceName it should not show it anymore if all is correct.
